core/io/image.cpp

```diff
@@ -2697,27 +2697,58 @@ void Image::blend_rect_mask(const Ref<Image> &p_src, const Ref<Image> &p_mask, c
 	}
 }
 
-void Image::fill(const Color &c) {
+// Makes chunk being repeated n times in consecutive memory.
+// Results in original chunk and (n-1) subsequent copies of it.
+#define COPY_CHUNK_REPEATEDLY_OVER_SUBSEQUENT_MEMORY(chunk_ptr, chunk_size, n)         \
```

Comment:
This could be turned into an online function, right?
Inline functions are preferred to macros, see https://github.com/godotengine/godot-proposals/issues/2093#issuecomment-757352474, https://github.com/godotengine/godot/pull/52071#discussion_r695164371 as an example

Reply:
Done. And thanks for the reviews!
I've added you as a co-author. :smiley: